bazelrc-preset.bzl

```diff
@@ -93,21 +93,23 @@ generate_preset = rule(
     },
 )
 
-def bazelrc_preset(name, out_file = None, **kwargs):
+def bazelrc_preset(name, out_file = None):
     """
     Creates a bazelrc preset file.
 
     Args:
         name: The name of the preset.
         out_file: The file to write the preset to. If not provided, the preset will be written to `{name}.bazelrc`.
-        **kwargs: additional named parameters to generate_preset
     """
     if lt("6.0.0"):
         fail("bazelrc_preset requires Bazel 6 or later. You are running Bazel {}".format(version))
     generate_preset(
         name = name,
         out = "_{}.bazelrc".format(name),
-        **kwargs
```

Comment:
note: this handling of kwargs was incorrect anyway, as it didn't allow tagging the diff_test for example.